<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1444</width>
    <height>758</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Image-Recognition</string>
  </property>
  <property name="styleSheet">
   <string notr="true">#MainWindow {
	background-color: white;
}</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="styleSheet">
    <string notr="true">QLabel {
	font-family: JUST Sans;
	font-size: 24px;
}

QLabel#prediction {
	color: rgb(255, 170, 0);
	font-weight: bold;
}

QLabel#accuracy {
	color: rgb(255, 0, 4);
	font-weight: bold;
}

QPushButton{
	font-family: JUST Sans;
	font-size: 20px;
}

QLineEdit {
	font-family: JUST Sans;
	font-size: 20px;	
	border-style: solid;
	border-width: 1px;
	border-radius: 10px 10px 10px 10px;
	color: rgb(0, 203, 0)
}

QFrame {
	background-color: white;
	border: 0px;
}

QLineEdit {
	font-family: JUST Sans;
	font-size: 20px;
}

QScrollArea QWidget{
	background-color: white;
}
</string>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QFrame" name="left_frame">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="sizePolicy">
       <sizepolicy hsizetype="MinimumExpanding" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>700</width>
        <height>660</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>0</width>
        <height>660</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <widget class="QLineEdit" name="img_path">
       <property name="geometry">
        <rect>
         <x>130</x>
         <y>10</y>
         <width>361</width>
         <height>40</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true"/>
       </property>
      </widget>
      <widget class="QPushButton" name="load_img">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>10</y>
         <width>110</width>
         <height>40</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true"/>
       </property>
       <property name="text">
        <string>Load</string>
       </property>
      </widget>
      <widget class="QPushButton" name="predict">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>60</y>
         <width>110</width>
         <height>40</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true"/>
       </property>
       <property name="text">
        <string>Predict</string>
       </property>
      </widget>
      <widget class="QFrame" name="hidden_frame">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>120</y>
         <width>681</width>
         <height>531</height>
        </rect>
       </property>
       <property name="frameShape">
        <enum>QFrame::StyledPanel</enum>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Raised</enum>
       </property>
       <widget class="QLabel" name="label_1">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>10</y>
          <width>121</width>
          <height>21</height>
         </rect>
        </property>
        <property name="text">
         <string>Prediction: </string>
        </property>
       </widget>
       <widget class="QLabel" name="label_3">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>45</y>
          <width>131</width>
          <height>31</height>
         </rect>
        </property>
        <property name="text">
         <string>Accuracy:</string>
        </property>
       </widget>
       <widget class="QLabel" name="accuracy">
        <property name="geometry">
         <rect>
          <x>130</x>
          <y>50</y>
          <width>512</width>
          <height>24</height>
         </rect>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
       <widget class="QLabel" name="prediction">
        <property name="geometry">
         <rect>
          <x>130</x>
          <y>10</y>
          <width>512</width>
          <height>24</height>
         </rect>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
       <widget class="QScrollArea" name="prediction_list">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>80</y>
          <width>361</width>
          <height>351</height>
         </rect>
        </property>
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="widgetResizable">
         <bool>true</bool>
        </property>
        <widget class="QWidget" name="list">
         <property name="geometry">
          <rect>
           <x>0</x>
           <y>0</y>
           <width>361</width>
           <height>351</height>
          </rect>
         </property>
        </widget>
       </widget>
      </widget>
     </widget>
    </item>
    <item>
     <widget class="QFrame" name="right_frame">
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QLabel" name="img">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Ignored" vsizetype="Ignored">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>700</width>
           <height>660</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true"/>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1444</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
